ingest/de.py

```diff
@@ -218,8 +255,8 @@ def run_h5ad(
 
         sc.pp.normalize_total(adata, target_sum=1e4)
         sc.pp.log1p(adata)
-        rank_key = "rank." + annotation + "." + method
         DifferentialExpression.de_logger.info(f"calculating DE")
+        rank_key = "rank." + annotation + "." + method
```

Comment:
An f-string would be a bit more locally-conventional.

```suggestion
        rank_key = f"rank.{annotation}.{method}"
```

Reply:
`296ceff`